docs/pages/database-access/reference/configuration.mdx

```diff
@@ -23,15 +23,26 @@ The following Proxy service configuration is relevant for database access:
   type="warning"
   title="Proxy TLS Warning for PostgreSQL"
 >
-  The PostgreSQL connection requires TLS enabled for the SSL connection that operates on the `web_listen_addr`.
-  Do not set `--insecure-no-tls` for the proxy Teleport instances as a parameter.  If you are terminating TLS at a
-  Application Load Balancer (ALB) or other service that may require enabling a backend protocol of HTTPS for the target address.  
+The following database types require TLS for SSL connections, using the
+`web_listen_addr` value:
+
+- Postgres
+- MongoDB
+- Redis
+- CockroachDB
+- Elasticsearch
+- OpenSearch
+
+Do not set `--insecure-no-tls` for the proxy Teleport
```

Comment:
What does "proxy" mean here? There are so many components of a Teleport setup that can be called a "proxy" that using this term alone can get confusing.